studybuddies/src/app/services/base-urlinterceptor.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import {HttpHandler, HttpRequest, HttpInterceptor} from '@angular/common/http';
+import {environment} from '../../environments/environment';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202842)
```suggestion
import {environment} from "../../environments/environment";
```